### PR TITLE
Testing Notebooks

### PR DESCRIPTION
**Pull request by HaoZeke**
As described in the readme and on the tutorial guide. Each tutorial is tested currently against the `conda` SymEngine.
image
Needs #1727 and #1726.

These get downloaded into the right location for [SymEngine.org](https://symengine.org) and displayed there.

----
This PR is part of the **Google Season of Docs 2020**

### CODE REVIEW

**Comment by HaoZeke** [comment]
@isuruf and @certik. More legible now.

File: binder/symedoctests.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: symengine_conda_doctests
```
Comment (isuruf) [inline binder/symedoctests.yml:1]: Why do we need this file?

**Review by isuruf — CHANGES_REQUESTED** [review]
Can you also remove the travis job that tests the notebooks?

File: binder/symedoctests.yml (hunk 1 above)
Comment (HaoZeke) [reply binder/symedoctests.yml:1]: We only really need `papermill` and `jupytext`, I'll just `conda install` them in GHA; thanks

Comment (HaoZeke) [reply binder/symedoctests.yml:1]: Would it be better to add to the other environment @isuruf? I notice it has `notebook` which isn't a build dependency...

**Comment by HaoZeke** [comment]
@isuruf done, sorry for the build spam. It'd be good to have this in quickly so I can setup the build from `symengine/symengine.github.io` (it'll need to copy these files over).

**Comment by rikardn** [comment]
@HaoZeke I have workflow for copying documentation here: https://github.com/pharmpy/pharmpy/blob/master/.github/workflows/deploy.yml

It took me a while to make the committer be the github bot.

**Review by certik — APPROVED** [review]
This looks good. Thanks for the work!

**Pull request merged**